controller/controller.go

```diff
@@ -201,6 +206,14 @@ func (c *Controller) Booted() chan struct{} {
 func (c *Controller) Reconcile(req reconcile.Request) (reconcile.Result, error) {
 	ctx := context.Background()
 
+	// Add common keys to the logger context.
+	{
+		loop := atomic.AddInt64(&c.loop, 1)
+
+		ctx = setLoggerCtxValue(ctx, loggerKeyController, c.name)
+		ctx = setLoggerCtxValue(ctx, loggerKeyLoop, strconv.FormatInt(loop, 10))
```

Comment:
Why `strconv.FormatInt`? 

```suggestion
		ctx = setLoggerCtxValue(ctx, loggerKeyLoop, int(loop))
```

Reply:
`strconv.FormatInt` produces string which is what `setLoggerCtxValue` takes.

Reply:
With you. 🙈